services/intersection-api/jpo-conflictvisualizer-api/README.md

```diff
@@ -0,0 +1,23 @@
+# Conflict Visualizer
+
+The Conflict Visualizer enables users to see events and assessments generated by the conflict monitor application. 
+
+
+# Dependencies
+
+The conflict Visualizer is dependent on the jpo-conflictmonitor, the jpo-geojsonconverter, and the jpo-ode.
+
+
+# Setup
+To setup the Conflict Monitor API, rename the application.properties.example file to application.properties
+Fill in any missing data in the file before running the API to ensure proper operation.
```

Comment:
question(blocking): should we include the mvnw commands in this section or is it documented somewhere else how to build and test this code?

Reply:
I am kind of a java noob - should I be specifying mvn or mvnw commands to build and run the api?